### PR TITLE
Add twitch info to streamers.json

### PR DESCRIPTION
**Pull request by luc-tielen**

### CODE REVIEW

File: streamers.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "channel": "https://www.twitch.tv/avh4",
     "speaking": ["EN"],
     "languages": ["Elm", "Haskell"],
-    "schedule": "[Various times Tu/Th/Su](https://twitter.com/avh4/status/1333478708934369282)"
+    "schedule":
```
Comment (chiroptical) [inline streamers.json:14]: Are these coming from an auto-formatter? I would rather these changes not be made

File: streamers.json [hunk 2]
```diff
@@ -88,5 +92,12 @@
     "speaking": ["EN"],
     "languages": ["Haskell"],
     "schedule": "Streams announced on [Twitter](https://twitter.com/tritlo)"
+  },
+  {
+    "name": "luctielen",
```
Comment (chiroptical) [inline streamers.json:97]: This isn't ideal, but could you maintain the alphabetical order? (I really should fix this in the README generator)

**Review by chiroptical — CHANGES_REQUESTED** [review]

**Review by chiroptical — APPROVED** [review]

**Comment by chiroptical** [comment]
Thanks @luc-tielen

File: streamers.json (hunk 1 above)
Comment (luc-tielen) [reply streamers.json]: Yes, sorry about that!

File: streamers.json (hunk 2 above)
Comment (luc-tielen) [reply streamers.json]: Wasn't aware of the order, fixed it :)

**Pull request merged**